collections.md

```diff
@@ -155,6 +155,151 @@ The json representation is as follow
 ```
 see also the [ResourcePolicies endpoint](resourcepolicies.md)
 
+### Collection Harvesting Settings
+**GET /api/core/collections/<:uuid>/harvester**
+
+It returns the harvesting settings for the current collection. This information is only accessible for users with collection administration permissions
+
+The harvest_type can be any of:
+* NONE
+* METADATA_ONLY
+* METADATA_AND_REF
+* METADATA_AND_BITSTREAMS
+
+The harvest_status can be any of:
+* READY
+* BUSY
+* QUEUED
+* OAI_ERROR
+* UNKNOWN_ERROR
+
+The metadata_config_id can be one of the ids from [Harvesting metadata configuration Endpoints](harvesting_metadata_configs.md)
```

Comment:
This link needs updating. It's now at `harvestermetadata.md`

Reply:
I've updated this now